ci/ci-test-tasks/upload-hotfix.yml

```diff
@@ -0,0 +1,22 @@
+parameters:
```

Comment:
Same with the hotfix - it is never used here. The hotfix functionality is used in another pipeline. We can get rid of it

Reply:
Will do